app/models/vacancy.rb

```diff
@@ -131,15 +131,6 @@ class Vacancy < ApplicationRecord
   extend VacancyFulTextSearchQueryModule
   scope :search_by_full_text, ->(query) { vacancy_full_text_search_query(query) }
 
-  # effectively we have two different types of vacancy - external ones and internal ones
```

Comment:
It seems a shame to lose this comment and the idea - IMHO ExternalVacancy should be a separate type, as it has vastly different validation rules (and even different fields)